### ISSUE TITLE
[Improvement]: Use Chome Headless to generate Page Previews

### ISSUE THREAD

**Issue by brusch** [pimcore/pimcore#10679]
### Improvement description

instead of wkhtmltoimage. 

wkhtmltoimage however should be still supported (to be removed in PXI) 

Example command: 

```
chromium --headless --disable-gpu --screenshot=/path/to/file.png --window-size=1200,2000 "http://pimcore.com/foo/bar"
```

we should check for `chromium` first and then for `chrome`. 

Could be just integrated here: 
https://github.com/pimcore/pimcore/blob/2f18a0f26380c56b609d937b40d2050c8d1a3e1a/lib/Image/HtmlToImage.php#L59

An also in the requirements check as an optional tool. 
I think we can just replace the check for wkhtmltoimage here.

**Comment by brusch** [comment]
Added to docker images by https://github.com/pimcore/docker/pull/67

**Comment by brusch** [comment]
For further tests we need to wait until https://github.com/php/php-src/pull/7981 was released with PHP 8.1.3

**Comment by brusch** [comment]
Maybe also related #11673
